### ISSUE TITLE
Add support for creating and revoking personal access tokens and fetching health check tokens.

### ISSUE THREAD

**Issue by gmessner** [gitlab4j/gitlab4j-api#336]
The GitLab API does not support the creation or revocation of personal access tokens, this makes it difficult to automate the testing of GitLab4J-API without a pre-configured and data seeded GitLab server running.

Adding the ability to create and revoke personal access tokens and fetch health check tokens will allow for the full automation of GitLab4J-API using the gitlab-ce docker image.

**Comment by gmessner** [comment]
This functionality has been released in 4.10.8

**Comment by baymac** [comment]
Thank you @gmessner for implementing this functionality.

**Comment by baymac** [comment]
I would like add a suggestion. If we can add static variable inside the AccessTokenUtils class so that the scope names can be clear. Avoids confusion for user.

For example,

```Java
public static final String API = "api";
public static final String READ_USER = "read_user";
public static final String READ_REPOSITORY = "read_repository";
...
```

I feel it could help users understand which variables to supply rather than checking them on GitLab Server. If you want to do this let me know, I will send PR. If you have a better suggestion also let me know. @gmessner

**Comment by gmessner** [comment]
@baymac 
I would prefer these to be an enum.  I am actually doing a release today so will add this enum.  Will let you know when it is done.

**Comment by baymac** [comment]
@gmessner yes, I found enum defined inside `Constants`. Presently I am using that but perhaps it could be improved. I would prefer enums inside `AccessTokenUtils`. Waiting for your release!

**Comment by gmessner** [comment]
@baymac 
The ```ApplicationScope``` is a different thing, I will be adding a ```Scope``` enum to the ```AccessTokenUtils``` class.  Should have this released shortly.

**Comment by baymac** [comment]
@gmessner I used it as
```Java
List<String> GL_PLUGIN_REQUIRED_SCOPE = ImmutableList.of(
    Constants.ApplicationScope.API.toValue(), // api
    Constants.ApplicationScope.READ_USER.toValue() // read_user
);
```
And it works like that. Btw what is the actual purpose of `ApplicationScope`?

**Comment by gmessner** [comment]
@baymac 
```ApplicationScope``` is for setting the scope of an OAUTH2 application, see ```ApplicationsApi```.

**Comment by baymac** [comment]
Got it. Thanks. @gmessner

**Comment by gmessner** [comment]
@baymac 
The ```Scope``` enum has been added to ```AccessTokenUtils``` and is available in release 4.11.4.

Just a heads up, the ```createAccessToken()``` method no longer takes List<String>, it is now List<Scope>

**Comment by baymac** [comment]
Awesome. :)
